assets/css/amp-validation-error-taxonomy.css

```diff
@@ -159,6 +159,27 @@ details[open] .details-attributes__summary::after, details.single-error-detail[o
 	margin-left: 10px;
 }
 
+body.taxonomy-amp_validation_error .wp-list-table .new th,
+body.taxonomy-amp_validation_error .wp-list-table .new td {
+	background-color: #fef7f1;
+}
+
+body.taxonomy-amp_validation_error .wp-list-table .new th.check-column {
+	border-left: 4px solid #d54e21;
+}
+
+body.taxonomy-amp_validation_error .wp-list-table .new th.check-column input {
+	margin-left: 4px;
+}
+
+.row-actions .amp_validation_error_accept > a {
+	color: #006505;
+}
+
+.row-actions .amp_validation_error_reject > a {
+	color: #a00;
+}
+
 .notice.accept-reject-error {
 	display: flex;
 }
```

Comment:
In a different CSS file, what do you think about changing this width to `20%`:

https://github.com/Automattic/amp-wp/blob/e67234d86375cbc3454416e84365219eab1d12df/assets/css/amp-validation-single-error-url.css#L118-L120

The 'status' icon sometimes appears above the `<select>` when the `<select>` is wider. Though this happens for all of them at slightly more narrow screen widths.

<img width="1437" alt="singel-url" src="https://user-images.githubusercontent.com/4063887/45888434-67422e00-bd94-11e8-887b-6dc9a3b133c1.png">

Reply:
This styling issue isn't caused by this PR, it's just more apparent when the `<select>` has longer text, like `'New Rejected'`.

Reply:
Applied in https://github.com/Automattic/amp-wp/pull/1429/commits/276c5a59495af6c3c2c1cfc3d4c484aa51083e88